railties/test/generators/app_generator_test.rb

```diff
@@ -590,6 +590,14 @@ def test_does_not_generate_system_test_files_if_skip_system_test_is_given
     end
   end
 
+  def test_html_lang_attribute_is_present
+    run_generator [destination_root]
+
+    assert_file "app/views/layouts/application.html.erb" do |contents|
+      assert_match(/<html lang="en"/, contents)
```

Comment:
Based on this test, it looks like the intention is for the app to be generated with a static locale? If that's the case, how would a user configure which locale to generate statically?

Reply:
Whoops. Sorry, I got myself confused! I forgot to escape the ERB tag!

Reply:
Fixed!